chore: update the description for storage_type

userzhangqg opened the pull request:
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description
Update the description of the supported storage types in the env example file.
Update the description of the storage config.


## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

userzhangqg committed:
feat: update the 'STORAGE_TYPE' supported in the env example and storage config description

crazywoola reviewed (changes requested):
We use `local` and `s3` by default.

crazywoola commented:
> > We use `local` and `s3` by default.
> 
> I just added the missing storage_type description

And you have changed the order as well :) the first 2 should be local and s3

userzhangqg committed:
chore: update storage type description

userzhangqg commented:
> > > We use `local` and `s3` by default.
> > 
> > 
> > I just added the missing storage_type description
> 
> And you have changed the order as well :) the first 2 should be local and s3

I understand, I modified it
